code/Pipfile

```diff
@@ -4,13 +4,13 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-flywheel-sdk = "==15.5.0"
-jinja2 = "*"
-flake8 = "*"
-flake8-colors = "*"
-pyyaml = "*"
-pydantic = "*"
-typing-extensions = "*"
+flywheel-sdk = "==15.8.0"
```

Comment:
Can we bump this up to a higher version, one that can use python 3.10 w/o any issues with collections. See https://flywheelio.zendesk.com/agent/tickets/7599

Reply:
Added